frontend/simple/js/state.js

```diff
@@ -99,19 +117,45 @@ const getters = {
   unreadMessageCount (state) {
     let mailboxContract = store.getters.mailboxContract
     return mailboxContract && mailboxContract.messages.filter((msg) => !msg.read).length
+  },
+  // Logged In user's identity contract
+  identity (state) {
+    return state.users[state.loggedIn]
+  },
+  // Logged In user's picture
+  picture (state) {
+    let identity = store.getters.identity
+    return identity && state[identity] && state[identity].attributes.picture
+  },
```

Comment:
Per above, delete. You can get this in other Vues by using the `currentUserContract` getter, and using vue-specific mixins like [`mapGetters`](http://vuex.vuejs.org/en/getters.html).